WebApi/BookService/LibraryList.cs

```diff
@@ -0,0 +1,216 @@
+﻿// <copyright file="LibraryList.cs" company="My company">
+//     Copyright (c) My company". All rights reserved.
+// </copyright>
+
+namespace WebApi.BookService
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using WebApi.Models;
+    using WebLib;
+
+    /// <summary>
+    /// Implementation of IBook
+    /// </summary>
+    public class LibraryList : ILibrary
```

Comment:
Strange name, maybe just  Library? or LibraryService etc

Reply:
I thought that this is good name because in the future I will write LibraryDatabase class.
If I would use LibraryService name, it will not be easy to understand is it local mock with data or class which works with database.
Am I right?